YoCode/TestCountCheck.cs

```diff
@@ -40,7 +42,7 @@ public void ExecuteTheCheck()
             tempStats = StatLine.GetNumbersInALine();
             StoreCalculations(tempStats);
 
-            UnitTestEvidence.FeatureImplemented = stats.percentagePassed == 100 ? true :false;
+            UnitTestEvidence.FeatureImplemented = stats.percentagePassed == 100 && stats.totalTests > TestCountTreshold ? true :false;
             UnitTestEvidence.GiveEvidence(StatLine);
             UnitTestEvidence.GiveEvidence("Percentage: "+ (stats.percentagePassed).ToString());
```

Comment:
Can we have a line of evidence showing `stats.totalTests` and `TestCountTreshold` as well?